SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc

```diff
@@ -1,6 +1,1072 @@
+
+|OBX-2
+|Value Type
+|
+|Set to *"NM"* since weight is always represented as a decimal number.
+
+|OBX-3/CWE-1
+|Identifier
+|
+|Set to MDC code *"68063"*.
+
+|OBX-3/CWE-2
+|Text
+|
+| Set to MDC RefId *"MDC_ATTR_PT_WEIGHT"*.
+
+|OBX-3/CWE-3
+|Name of Coding System
+|
+|Set to coding system *"MDC"*.
+
+|OBX-4
+|Observation Sub-ID
+|
+|Set to *"<MDS>.0.0.2"* where *<MDS>* is the number of the MDS level assigned by the gateway.
+See <<ref_dec_obx4>> for further information.
+
+|OBX-5
+|Observation Value
+|pm:PatientContextState
+/pm:CoreData
+/pm:Weight
+/@MeasuredValue
+|Note that the decimal number needs to be formatted according to the HL7 numeric value formatting rules.
+
+|OBX-6
+|Units
+|
+|HL7 data type *CWE*
+
+|OBX-6/CWE-1
+|Identifier
+|pm:PatientContextState
+/pm:CoreData
+/pm:Weight
+/pm:MeasurementUnit
+/@Code
+|
+
+|OBX-6/CWE-2
+|Text
+|If @Code is an MDC code, this field contains the RefId of the MDC code.
+
+In all other cases, the field is set to the pm:PatientContextState
+/pm:CoreData
+/pm:Weight
+/pm:MeasurementUnit
+/@SymbolicCodeName.
+|Note that MDC is the default coding system if no coding system is specified.
+
+|OBX-6/CWE-3
+|Name of Coding System
+|*"MDC"* if no other coding system is specified.
+
+In all other cases, the field is set to pm:PatientContextState
+/pm:CoreData
+/pm:Weight
+/pm:MeasurementUnit
+/@CodingSystem.
+|Note that MDC is the default coding system if no coding system is specified.
+
+|OBX-11
+|Observation Result Status
+|
+|When the patient context has been associated and a new @BindingStartTime has been set, the field is set to final result status "F".
+
+When there are further updates of the weight value after the association of the patient context, the field is set to "C".
+
+|OBX-14
+|Date/Time of the Observation
+|pm:PatientContextState
+/@BindingStartTime
+|Note that the HL7 date & time format differs from the xsd date/time formats and requires a mapping accordingly (see also <<ref_expl_dt_mapping>>).
+
+|===
+
+include::tf2-ch-b-gateway-pv1-mapping.adoc[]
+
+===== OBR - Observation Request Segment
+The the HL7 Obervation Request (OBR) segment requires a mapping from the SDC containment tree and metric data to the OBR segment fields.
+
+====== OBR-2 Placer Order Number
+
+.R8003
+[sdpi_requirement#r8003]
+****
+
+For the IHE DEC profile, the <<actor_somds_dec_gateway>> shall set the OBR-2 field to the identifier of the Device Observation Reporter (DOR) of the IHE DEC gateway (not the individual device identifier).
+
+.Notes
+[%collapsible]
+====
+NOTE: For further information, please refer to the <<ihe_pcd_tf-2_2019>>.
+====
+****
+
+====== OBR-3 Filler Order Number
+
+.R8004
+[sdpi_requirement#r8004]
+****
+
+For the IHE DEC profile, the <<actor_somds_dec_gateway>> shall set the OBR-3 field to the identifier of the Device Observation Reporter (DOR) of the IHE DEC gateway (not the individual device identifier).
```

Comment:
Shall this be the exact same information as Placer Order Number? 
Shouldn't be one Device Observation Reporter and one Device Observation Consumer?

Reply:
The DOR has no information about the consumers. Therefore, the IHE TF defines that the Placer and Filler field are identical.

Reply:
IHE TF-2 says "OBR-2 Placer Order Number will usually be empty given that this is an unsolicited order.". I would prefer to leave the Placer Order Number empty.